### DISCUSSION TITLE
feature: tab outline

### DISCUSSION THREAD

**Discussion by Tom-ahawk** [kinghat/tabbed-card#32] [Feature Requests]
Please consider making the outline for the card, and the active tab, more like the traditionally tab outline. 

![image](https://user-images.githubusercontent.com/35527507/200117537-d1bae7f8-64e5-4e48-940e-f6137098b60e.png)

**Comment by kinghat** [comment]
the underlying tab component, powered by material web components, has its own feature set which outlines are not apart of. these components are what home assistant themselves use. i wont say that this feature request will never be implemented but the goal is to keep the components as stock/standard as possible to align with home assistant. you can see the current component styles and attributes here:

https://material-components.github.io/material-web/demos/tabs/

https://github.com/material-components/material-web/blob/mwc/packages/tab-bar/README.md

**Comment by kinghat** [comment]
referencing similar request from elsewhere: https://github.com/kinghat/tabbed-card/issues/83

**Reply by ildar170975** [reply]
![изображение](https://user-images.githubusercontent.com/71872483/218887754-1522c5c3-96e4-4ab1-b3df-6481d18948d4.png)

```
            mwc-tab {
              background: var(--ha-card-background, var(--card-background-color, white) );
              border-color: var(--ha-card-border-color, var(--divider-color, #e0e0e0) );
              border-width: 1px;
              border-top-left-radius: 12px;
              border-top-right-radius: 12px;
              border-style: solid;
              overflow: hidden;
            }
            section article > * {
              --ha-card-border-radius: 0px 0px 12px 12px;
            }
```

**Reply by ildar170975** [reply]
Check here also: https://community.home-assistant.io/t/card-mod-add-css-styles-to-any-lovelace-card/120744/4280
